src/global/project/mod.rs

```diff
@@ -373,23 +389,29 @@ mod tests {
         python = "python"
```

Comment:
Can we change this simple manifest to point to a local channel? That way the tests will be significantly faster.